### PR TITLE
PS-5371 Rhel8 changes

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build-ps/percona-server.spec [hunk 1]
```diff
@@ -483,8 +484,11 @@ mkdir debug
            -DWITH_ROCKSDB=1 \
            -DWITH_INNODB_MEMCACHED=1 \
            -DWITH_ZLIB=system \
+           -DWITH_NUMA=ON \
            -DWITH_SCALABILITY_METRICS=ON \
            -DMYSQL_MAINTAINER_MODE=OFF \
+           -DFORCE_INSOURCE_BUILD=1 \
+           -DWITH_NUMA=ON \
```
Comment (hors) [inline build-ps/percona-server.spec:490]: you use the same option twice

File: build-ps/percona-server.spec [hunk 2]
```diff
@@ -521,8 +525,11 @@ mkdir release
            -DWITH_ROCKSDB=1 \
            -DWITH_INNODB_MEMCACHED=1 \
            -DWITH_ZLIB=system \
+           -DWITH_NUMA=ON \
```
Comment (hors) [inline build-ps/percona-server.spec]: please see comment above

File: build-ps/percona-server.spec (hunk 1 above)
Comment (EvgeniyPatlan) [reply build-ps/percona-server.spec:490]: fixed. thanks

File: build-ps/percona-server.spec (hunk 2 above)
Comment (EvgeniyPatlan) [reply build-ps/percona-server.spec]: fixed. thanks

**Review by hors — APPROVED** [review]

**Pull request merged**